### DISCUSSION TITLE
can you combine mcp reborn with fabric

### DISCUSSION THREAD

**Discussion by caidenmccarthy** [Hexeption/MCP-Reborn#193] [Q&A]
is it possible to combine this with fabric or forge so i can edit the source code while also creating new things with forge/fabric

**Comment by APie357** [comment]
That is not currently possible. MCP is a tool to look at Minecraft's source code and modify it. Since it modifies code that is expected to be a certain value by modloaders, it isn't compatible. If you would like to use fabric mods with your mods, you will need to make your mod for fabric.
